### PR TITLE
feat(security): encrypt login password

### PR DESCRIPTION
**Pull request by baurine**
## What Did

Encrypt the SQL Auth password in the frontend side by the RSA public key, and decrypt it in the backend side.

## Preview

image

image

### CODE REVIEW

**Comment by mornyx** [comment]
Backend LGTM. BTW, Should we provide a configuration item for this feature (maybe disabled by default), rather than as the only choice?

File: ui/packages/tidb-dashboard-for-op/src/dashboardApp/layout/signin/index.tsx [hunk 1]
```diff
@@ -517,6 +540,7 @@ function App({ registry }) {
             <TiDBSignInForm
               successRoute={successRoute}
               onClickAlternative={handleClickAlternative}
+              publicKey={publicKey}
```
Comment (shhdgit) [inline ui/packages/tidb-dashboard-for-op/src/dashboardApp/layout/signin/index.tsx:543]: Why not pass `publicKey={loginInfo.sql_auth_public_key}` directly?

**Review by shhdgit — APPROVED** [review]
LGTM.

**Comment by baurine** [comment]
> Backend LGTM. BTW, Should we provide a configuration item for this feature (maybe disabled by default), rather than as the only choice?

After we can encrypt the login password, I don't think the user want to see the plain password again.

**Pull request merged**